pkg/kubernetes/kubernetes.js

```diff
@@ -125,6 +125,24 @@ define([
             console.warn(ex);
         }
 
+        this.delete_pod = function delete_pod(pod_name) {
+            api.request({"method": "DELETE",
+                "params": "",
+                "path": "/api/v1beta1/pods/" + pod_name,
+                "body": "",
+                "headers": ""
```

Comment:
Ditto for headers. Suggest removing this line. In any case ```"headers"``` should be JSON object, not a string.